### PR TITLE
UHM-7470, remove dossier column if identifier not defined

### PR DESCRIPTION
**Pull request by cioan**

### CODE REVIEW

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @cioan !

File: omod/src/main/webapp/pages/reports/inpatientList.gsp [hunk 1]
```diff
@@ -80,8 +82,10 @@
     %>
     <tr id="visit-${ v.patientId
     }">
-        <td>${ v.zlEmrId ?: ''}</td>
-        <td>${ v.dossierNumber ?: ''}</td>
+        <td>${ v.zlEmrId ?: v.primaryIdentifier }</td>
```
Comment (mseaton) [inline omod/src/main/webapp/pages/reports/inpatientList.gsp:85]: I'm pretty sure the zlemrid _is_ the primary identifier in Haiti, right?  Or is this sometimes different?  If it is the same, can't we just use primaryIdentifier everywhere?

**Review by mseaton — APPROVED** [review]
LGTM, though see my comment (I'm fine just merging this as-is though)

File: omod/src/main/webapp/pages/reports/inpatientList.gsp (hunk 1 above)
Comment (cioan) [reply omod/src/main/webapp/pages/reports/inpatientList.gsp]: You are correct, @mseaton ! I just replaced the zlEmrId with the primaryIdentifier. Thanks!

**Pull request merged**